.travis.yml

```diff
@@ -21,6 +21,7 @@ script:
   # Unit tests
   - python3 -m tokenizers.parsers
   - python3 -m unittest tokenizers.block_tokenizer_tests
+  - python3 -m unittest tokenizers.function_extractor_tests
```

Comment:
You can simplify tests - use `python -m unittest discover` https://docs.python.org/3/library/unittest.html#unittest-test-discovery - it will automatically launch all tests